src/scripts/setup.py.in

```diff
@@ -53,7 +53,7 @@ setup(
     url                  = 'http://triton.quarkslab.com',
     zip_safe             = False,
     include_package_data = True,
-    data_files           = [([path for path in site.getsitepackages() if 'System' not in path][0], TRITON_MODULES)], # NOTE: Must be cleaner than this hack.
+    data_files           = TRITON_MODULES,
```

Comment:
We got [warning](https://travis-ci.org/JonathanSalwan/Triton/jobs/191905345#L1274)